### PR TITLE
Rewmove isAutomated flag from systemtest build file

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Rewmove isAutomated flag from systemtest build file + resolves https://github.com/AdoptOpenJDK/openjdk-tests/issues/693
Signed-off-by: Mesbah Alam <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
These flags are not required any more, due to : https://github.com/AdoptOpenJDK/openjdk-systemtest/pull/194

**Comment by Mesbah-Alam** [comment]
Tested with: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/434/console

File: systemtest/build.xml [hunk 1]
```diff
@@ -64,8 +64,9 @@
 			<arg value="clone" />
 			<arg value="--depth" />
 			<arg value="1" />
-			<arg value="--single-branch" />
-			<arg value="${git-prefix}://github.com/AdoptOpenJDK/openjdk-systemtest.git" />
+			<arg value="-b" />
+			<arg value="downloadPrereqJarIfNotCopied"/>
+			<arg value="${git-prefix}://github.com/Mesbah-Alam/openjdk-systemtest.git" />
```
Comment (sophia-guo) [inline systemtest/build.xml]: those are still under your own branch

Comment (Mesbah-Alam) [reply systemtest/build.xml]: Yes, I was still running some tests builds. Removed now :)

**Comment by Mesbah-Alam** [comment]
Note: this PR contains the changes recommended in https://github.com/AdoptOpenJDK/openjdk-tests/issues/693 as well.

File: buildenv/jenkins/JenkinsfileBase [hunk 2]
```diff
@@ -193,12 +193,18 @@ def buildTest() {
...
-				//get pre-staged test jars from systemtest.getDependency build before system test compilation
-				copyArtifacts fingerprintArtifacts: true, projectName: "systemtest.getDependency", selector: lastSuccessful(), target: 'openjdk-tests/TestConfig/lib'
 			} catch (Exception e) {
 				echo 'Cannot run copyArtifacts from test.getDependency. Skipping copyArtifacts...'
 			}
+			
+			try { 
+				if (env.BUILD_LIST == 'systemtest') { 	
+					//get pre-staged test jars from systemtest.getDependency build before system test compilation
+					copyArtifacts fingerprintArtifacts: true, projectName: "systemtest.getDependency", selector: lastSuccessful(), target: 'openjdk-tests'
```
Comment (llxia) [inline buildenv/jenkins/JenkinsfileBase]: Should we download into `openjdk-tests/systemtest_prereqs`?

Comment (llxia) [reply buildenv/jenkins/JenkinsfileBase]: Sorry, did I miss somehting? The code here means we download it to `openjdk-tests`.

Comment (Mesbah-Alam) [reply buildenv/jenkins/JenkinsfileBase]: We are copying `systemtest_prereqs` folder from systemtest.getDependency build and placing it under openjdk-tests. https://github.com/AdoptOpenJDK/openjdk-tests/blob/1114a6936698ee0cd4adb86beb95b866837ebd6d/buildenv/jenkins/getDependency#L26

Comment (llxia) [reply buildenv/jenkins/JenkinsfileBase]: ah, thanks.

**Pull request merged**